MarkdownView/markdownview/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="us.feras.mdv">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
```

Comment:
Cool! Yeah no worries, I really like this idea though and I was surprised to see there are very little proper solutions to render markdown on Android. I attempted Pegdown, but it has no Android support (only Java as it uses some fancy reflection features) and Bypass, which does not process links or images correctly. Nice job on this!